src/reducers/poiForm.reducer.js

```diff
@@ -54,6 +57,18 @@ export default function poiForm(state = initialState.poiForm, action) {
         ...state,
         media: [...state.media].filter(media => media.id !== action.payload.id)
       }
+    case POI_COPIED:
```

Comment:
if you intentionally left this blank, put a `TODO` comment or something. otherwise it'll have the same behavior as `POI_FORM_POI_PASTED`